### ISSUE TITLE
Session & Speaker: User can change his session/speaker details after CFS is closed

### ISSUE THREAD

**Issue by kushthedude** [fossasia/open-event-frontend#3768]
**Describe the bug**

User can change his session/speaker details after CFS is closed.

**To Reproduce**

Steps to reproduce the behaviour:

1. Submit a Proposal and Confirm it.
2. Wait for CFS to get closed and then edit your proposal.
3. See error

**Expected behaviour**

User should not have the right to edit the proposal once CFS is ended.

**Comment by maze-runnar** [comment]
@kushthedude i want to work on this .

**Comment by iamareebjamal** [comment]
This is not a frontend bug. Open an issue on the server

**Comment by kushthedude** [comment]
Server should restrict the editing and we can hide the options on FE after
cfs is closed.

**Comment by iamareebjamal** [comment]
Yes, that is an enahncement of the frontend. The bug still needs to solve on backend

**Comment by maze-runnar** [comment]
@kushthedude i want to work on this.

**Comment by mariobehling** [comment]
Looking into this again, I see that this is not an issue. We already resolved this as we added the option to "lock a session" into the session area.

![Screenshot from 2020-07-18 11-23-52](https://user-images.githubusercontent.com/1583873/87849470-46dc4f80-c8e9-11ea-994c-7ed140b5d250.png)

Therefore closing this issue. We need to test in detail if the "lock session" feature is working everywhere as expected though.
